doc/wikis/cereal.md

```diff
@@ -0,0 +1,204 @@
+## SHOGUN `Cereal` serialization framework
+
+#### Table of Contents
+
+- [Motivation](#motivation)
+- [For SHOGUN developers](#For SHOGUN developers)
+  - [Examples] (#Examples)
+- [For serialization framework developers] (#For serialization framework developers)
+  - [Serialization interface] (#Serialization interface)
+  - [Serialization methods in `SGObject`] (#Serialization methods in `SGObject`)
+  - [Serialization methods in `Any`] (#Serialization methods in `Any`)
+  - [Serialization methods in `SGVector`, `SGMatirx` and `SGReferencedData`] (#Serialization methods in `SGVector`, `SGMatirx` and `SGReferencedData`)
+
+
+### Motivation
+
+[`Cereal`](http://uscilab.github.io/cereal/) is a header-only C++11 serialization library that is fast, light-weight, and easy to extend. 
+
+The `Cereal` serialization framework in SHOGUN uses the new tag parameter framework, which allows the easy and readable archive of `SGObject` data.
```

Comment:
no need of 'the' before easy: 'which allows easy and readable archive....'.
`SGObject` can be `CSGObject` or `SGObject.h` everywhere.